PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -20,7 +20,7 @@ protocol SelectableViewController: UIViewController {
     func didBecomeSelected(by parent: SavesContainerViewController)
 }
 
-class SavesContainerViewController: UIViewController, UISearchBarDelegate {
+class SavesContainerViewController: UIViewController, UISearchBarDelegate, UITextFieldDelegate {
```

Comment:
why do we need to add this delegate?

Reply:
needed it for an older solution i had for it. removed now.